UI/tests/common/mocks/lsmb_elements.js

```diff
@@ -1,4 +1,7 @@
 /** @format */
+/* eslint-disable vue/component-definition-name-casing */
+/* eslint-disable vue/require-prop-types, vue/one-component-per-file */
```

Comment:
require-prop-types is globally disabled, no?

Reply:
Yes, for vue files

Reply:
What do you mean by that? Why does this file want to report a vue error in the first place? It's not a Vue file? (Can't we disable Vue checks for non-Vue files?)

Reply:
`require-prop-types` is only globally disabled in the vue section, not in the js one. `lsmb_elements.js`  is a vue component